templates/registration/register.html

```diff
@@ -2,21 +2,38 @@
 {% load staticfiles %}
 
 {% block content %}
-    <h2 class="extra-padding">Register</h2>
-    <div class="break">
-        <hr>
-    </div>
-    <div class="center">
-        <p>Sign up with one click using Facebook or Google</p>
-        <a href="{% url 'social:begin' backend='facebook-register' %}"><img class="oauth-icon" src="{% static "img/facebook-signup.png" %}" alt="Register with Facebook" /></a><br/>
-        <a href="{% url 'social:begin' backend='google-oauth2-register' %}"><img class="oauth-icon" src="{% static "img/google-signup.png" %}" alt="Register with Google" /></a>
-    </div>
-    <div class="form-input">
-        <p>or create an account the old-fashioned way</p>
-        <form action="" method="POST">
-            {% csrf_token %}
-            {{ form.as_p }}
-            <input class="button submit-button" type="submit" value="Register" />
-        </form>
+    <div id="register-page" class="page">
+        <h2 class="extra-padding">Register</h2>
+        <div class="break">
+            <hr>
+        </div>
+        <div class="register-buttons">
+            <div class="button-container">
+                <div class="sign-up-button facebook">
+                    <h5 class="button-font"><a class="facebook-font" href="{% url 'social:begin' backend='facebook-register' %}"><i class="fa fa-facebook button-icon-padding" aria-hidden="true"></i>Sign Up with Facebook</a></h5>
+                </div>
+            </div>
+            <div class="button-container">
+                <div class="sign-up-button google">
+                    <h5 class="button-font"><a class="google-font" href="{% url 'social:begin' backend='google-oauth2-register' %}"><i class="fa fa-google button-icon-padding" aria-hidden="true"></i>Sign Up with Google</a></h5>
+                </div>
+            </div>
+
+            <div class="button-container">
+                <div class="sign-up-button manually">
+                    <h5 class="button-font"><a class="manual-font" data-open="signup-modal"><i class="fa fa-envelope button-icon-padding" aria-hidden="true"></i>Sign Up with e-mail</a></h5>
+                </div>
+            </div>
+            <div class="reveal center" id="signup-modal" data-reveal>
+                <form action="" method="POST">
+                    {% csrf_token %}
+                    {{ form.as_p }}
+                <input class="button submit-button" type="submit" value="Register" />
```

Comment:
This line should be indented in, since the `<input>` button is inside the form:
```html
<form action="" method="POST">
    {% csrf_token %}
    {{ form.as_p }}
    <input class="button submit-button" type="submit" value="Register" />
</form>
```